docs/prologue/introduction.md

```diff
@@ -55,7 +55,7 @@ The project has guiding goals for its development:
 
 ## Chat with Perses project team?
 
-You can find the project team on Matrix in the #perses-dev channel, see the [contributing guide](https://github.com/perses/perses/blob/main/CONTRIBUTING.md)
+You can get in touch with the project team on the development channel [#perses-dev](https://cloud-native.slack.com/messages/C07KQR95WBE) on [CNCF slack](https://slack.cncf.io/), see the [contributing guide](https://github.com/perses/perses/blob/main/CONTRIBUTING.md)
```

Comment:
`CONTRIBUTING.md` was already updated with the link to the new CNCF Slack channel. Better to keep a "single source of truth" for such links I guess to avoid future potential desync.

I like how you rephrased the sentence here though. Could you keep just this part?
```suggestion
You can get in touch with the project team on the development channel, see the [contributing guide](https://github.com/perses/perses/blob/main/CONTRIBUTING.md)
```